pkg/templates/templates.go

```diff
@@ -84,8 +81,10 @@ var (
 type Config struct {
 	AdditionalIndentation uint
 	AESKey                []byte
+	DecryptionConcurrency uint8
+	DecryptionEnabled     bool
 	DisabledFunctions     []string
-	EncryptionMode        EncryptionMode
+	EncryptionEnabled     bool
```

Comment:
Is there a reason `EncryptionEnabled` and `DecryptionEnabled` are separate variables? Logically if encryption is enabled would decryption need to be as well?

Reply:
@pshickeydev in our use-case, they would not be enabled at the same time, but it's an easier API interface than an encryption mode enum. I suppose a consumer of this library may want to decrypt some content and reencrypt it, but we don't have a use-case for it.